tools/spice/spice_examples.py

```diff
@@ -0,0 +1,132 @@
+"""Annotated Events: project events in instrument frame to reference systems.
+
+Reference: https://spiceypy.readthedocs.io/en/main/documentation.html.
+"""
+
+import logging
+
+import spiceypy as spice
+
+from tools.spice.spice_utils import list_all_constants
+
+# Logger setup
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def get_attitude_timerange(ck_kernel, id):
+    """Get attitude timerange using the ck kernel.
+
+    Parameters
+    ----------
+    ck_kernel: str
+        Directory of the ck kernel.
+    id: int
+        ID of the instrument or spacecraft body.
+
+    Returns
+    -------
+    start: float
+        Start time in ET.
+    end: float
+        End time in ET.
+    """
+    # Get the first CK path
```

Comment:
Can you also define the acronyms used in the comments/docs? I think it's ok to leave "CK" as a common acronym in the variable names here, but it should be made clear in the docs what the "CK Kernel" is used for and contains.